### PR TITLE
feat: 英文化Animator文档 & 增加AnimationEvent Playground

### LINKED ISSUE #202: 【Englishization】骨骼动画

### PR DESCRIPTION
**Pull request by luzhuang**
Close #202

### CODE REVIEW

File: docs/animator.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+<playground src="skeleton-animation-crossFade"></playground>
+
+You can use the [crossFade](${api}core/animation/Animator#crossFade) method to make the character transition to a specified state. The first parameter is the `name` of the animation state to be transitioned to, and the second parameter is the normalized animation transition time. For the description of other parameters, please refer to [API document](${api}core/animation/Animator#crossFade ).
+
+### Add events to animation
+
+<playground src="animation-event"></playground>
+
+You can use [AnimatioonEvent](${api}core/animation/AnimationEvent) to add events to AnimationClip. The animation event will call the specified callback function of the component you bind to the same entity at the specified time.
```
Comment (yangfengzzz) [inline docs/animator.md:61]: AnimationEvent

File: docs/animator.zh-CN.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+<playground src="skeleton-animation-crossFade"></playground>
+
+你可以使用 [crossFade](${api}core/animation/Animator#crossFade) 方法来使角色过渡到指定状态。第一个参数为要过渡到的动画状态的`name`，第二个参数为归一化的动画过渡时间，其他参数说明详见[API文档](${api}core/animation/Animator#crossFade)。
+
+
+### 动画事件
+
+<playground src="animation-event"></playground>
+
+你可以使用 [AnimatioonEvent](${api}core/animation/AnimationEvent) 来为AnimationClip添加事件，动画事件将在指定时间调用你在同一实体上绑定组件的指定回调函数。
```
Comment (yangfengzzz) [inline docs/animator.zh-CN.md:64]: AnimationEvent

**Review by yangfengzzz — CHANGES_REQUESTED** [review]

File: docs/animator.md (hunk 1 above)
Comment (luzhuang) [reply docs/animator.md]: fixed

File: docs/animator.zh-CN.md (hunk 2 above)
Comment (luzhuang) [reply docs/animator.zh-CN.md]: fixed

**Review by zhuxudong — APPROVED** [review]
+1

**Pull request merged**